Use NULL instead of TAILQ_END().

TAILQ_END() is not defined on GNU/Linux; on OpenBSD, it is defined to
be NULL. Replace the macro with a NULL to make things work.
Index: trunk/scrotwm.c
===================================================================
--- trunk.orig/scrotwm.c	2009-10-13 15:22:54.000000000 +0200
+++ trunk/scrotwm.c	2009-10-13 16:05:57.000000000 +0200
@@ -3767,7 +3767,7 @@
 	ws = win->ws;
 	wl = &ws->winlist;
 
-	for (w = TAILQ_FIRST(&ws->winlist); w != TAILQ_END(&ws->winlist); w = wn) {
+	for (w = TAILQ_FIRST(&ws->winlist); w != NULL; w = wn) {
 		wn = TAILQ_NEXT(w, entry);
 		if (win == w)
 			continue; /* can't happen but oh well */
